test/e2e/tests/test_notebook_instance.py

```diff
@@ -84,6 +86,7 @@ def get_notebook_instance_resource_status(reference: k8s.CustomResourceReference
     return resource["status"]["notebookInstanceStatus"]
 
 
+@flaky
```

Comment:
Great! how do you set num of retires? is there a default?

Reply:
we can set it like this `@flaky(max_runs=3, min_passes=2)`

default is `max_runs=2, min_passes=1`